concert/app/src/functions/on_event_stage_changed/index.py

```diff
@@ -88,3 +104,107 @@ def get_song_list_update_transaction_item(message_data):
     }
 
     return {"Update": song_list_update_args}
+
+
+def get_song_list_get_transaction_item():
+    serializer = boto3.dynamodb.types.TypeSerializer()
+    song_list_get_args = {
+        "Key": {ATTR_RECORD_ID: serializer.serialize(RECORD_ID_SONG_LIST)},
+        "TableName": DYNAMODB_TABLE_NAME,
+    }
+
+    return {"Get": song_list_get_args}
+
+
+def get_aggregate_data_get_transaction_item():
+    serializer = boto3.dynamodb.types.TypeSerializer()
+    aggregate_data_get_args = {
+        "Key": {ATTR_RECORD_ID: serializer.serialize(RECORD_ID_AGGREGATE)},
```

Comment:
These two functions are identical except for a single argument. DRY opportunity.

Reply:
Done.